src/json.hpp

```diff
@@ -5205,7 +5162,7 @@ class binary_reader
     @brief get next character from the input
 
     This function provides the interface to the used input adapter. It does
-    not throw in case the input reached EOF, but returns
+    not throw in case the input reached EOF, but returns a -'ve valued
```

Comment:
We should remove this "negative valued" bit, right?

Reply:
Agreed.